Core/LocalSettings.cs

```diff
@@ -70,17 +70,24 @@ public class Mqtt
 		public string TopicKmlListOfActiveObj { get; set; } = "KMLNode/ActiveKMLObjects";
 
 	}
-
-
+	public class ShowMode
```

Comment:
I think i would rather have class named SpeedLimiter and with the property being e.g. Enabled, and MaxSpeed

Reply:
Also i think u have spaces configured in VS2022 and its sometimes spaces, sometimes tabs... Lets just use tabs.

Reply:
Done in [913e6dc](https://github.com/wisniax/RoverControlApp/pull/16/commits/913e6dc6c973d47e15a13d6f8d0d4971c55f4eeb) and [5cdd699](https://github.com/wisniax/RoverControlApp/pull/16/commits/5cdd69927233201a74920a5928bac8b26934d07b) (replaced those that could find). I didn't notice "keep tabs" wasn't on by default